### PR TITLE
(.gitignore) Add workaround for atom fuzzy-finder index: https://github.com/atom/fuzzy-finder/issues/65

### PR DESCRIPTION
**Pull request by mrvisser**

### CODE REVIEW

**Comment by mrvisser** [comment]
Assigned to @simong for review

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,11 @@
 .DS_STORE
 dump.rdb
 target/
+*.log
+doc_data.txt
+
 node_modules/*
 !node_modules/oae-*
-node_modules/oae-rest
+!node_modules/oae-*/*
```
Comment (simong) [inline .gitignore]: Why is this required? Wouldn't `!node_modules/oae-*` cover this?

Comment (mrvisser) [inline .gitignore]: For git, yes. But this is the atom bug workaround.

Comment (simong) [inline .gitignore]: Can you add a comment as it not immediately obvious and an oaf like me might remove it when trying to micro optimize?

**Comment by mrvisser** [comment]
I accidentally pushed this fix to master with a README.md change. I've followed up on this here: https://github.com/oaeproject/Hilary/commit/9711b11eb93583a9b73a03be8fb326a937dd5463
